docs/docs/keymaps/input-processors/behaviors.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Behaviors Input Processor
+sidebar_label: Behaviors
+---
+
+## Overview
+
+The behaviors input processor is used invoke standard behaviors when certain input events occur; most frequently this is used to trigger behaviors when certain mouse buttons are triggered by physical pointing devices. For split devices, this is used with listeners on the central, and will never invoke behaviors on the split peripheral devices.
```

Comment:
I think this is a bit imprecise when we consider global behaviors, and that they will do something on the peripherals. I like the wording in the note box in https://zmk.dev/docs/keymaps/combos#advanced-usage better.

Reply:
Fair. Added a note that mimicked that wording.